brayns/engine/Engine.cpp

```diff
@@ -218,6 +238,16 @@ Renderer &Engine::getRenderer() noexcept
     return *_renderer;
 }
 
+EngineObjectFactory<Camera> &Engine::getCameraFactory() noexcept
```

Comment:
Maybe using `EngineObjectFactory<Camera> = CameraFactory;` somewhere (and renderer).

Reply:
As personal preference I prefer not to hide the real type with `using`, unless is too long